packages/features/ee/teams/components/MemberInvitationModal.tsx

```diff
@@ -59,8 +59,8 @@ interface FileEvent<T = Element> extends FormEvent<T> {
   target: EventTarget & T;
 }
 
-function toggleElementInArray(value: string[] | string, element: string): string[] {
-  const array = Array.isArray(value) ? value : [value];
+function toggleElementInArray(value: string[] | string | undefined, element: string): string[] {
+  const array = value ? (Array.isArray(value) ? value : [value]) : [];
```

Comment:
[Sometimes error comes when adding an existing user to an org ](https://www.loom.com/share/227f966d5541413588ee589b0e460de7)

cc @sean-brydon I don't know why it has been passed an undefined value, TypeScript isn't complaining. But this changes fixes the issue.